win32ss/user/user32/misc/exticon.c

```diff
@@ -283,7 +285,19 @@ static UINT ICO_ExtractIconExW(
 	UINT cxDesired,
 	UINT cyDesired,
 	UINT *pIconId,
-	UINT flags)
+#ifdef __REACTOS__
+    UINT flags,
+    /* This function is called from two difference code paths.
+     * One is from Shell32 using the ExtractIconEx function.
+     * The other is from User32 using PrivateExtractIcons.
+     * Based on W2K3SP2 testing, the count of icons returned
+     * is zero (0) for PNG ones using ExtractIconEx and
+     * one (1) for PNG icons using PrivateExtractIcons. 
+     * We can handle the difference using the fUser32 flag.*/
+    BOOL fUser32)
```

Comment:
calling that "user32" is a misnomer because  both cases are called within.... user32. It's just that one (TRUE) is called from within PrivateExtractIconsW, while the other one (FALSE) is called from within PrivateExtractIcon***Ex***W.

Reply:
What name would you suggest for the flag @hbelusca?

Reply:
Maybe something like `fIconEx` .

Reply:
Changed flag name from 'fUser32' to 'fIconEx'. Thanks.